README.md

```diff
@@ -0,0 +1,13 @@
+## Updating library version in package-lock.json.
+
+Since the version of dependency `@orchestrator-ui/orchestrator-ui-components` is `*` for proper usage in our Turborepo, we still need to update package-lock.json to make the build command build the app with the latest library version.
```

Comment:
For a user that's unfamiliar with the library repo this is confusing I think. Also it only applies if you have already have  a previous installation. I would remove this or expand it to include those conditions

Reply:
Renaming the file to have it not shown direct;y when visiting the repository's webpage